### PR TITLE
Updated german translations

### PR DESCRIPTION
**Pull request by GuepardoApps**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -54,15 +54,15 @@
...
+        Ein einfacher Musik-Player, der ganz einfach über die Benachrichtigungsleiste, das Homescreen Widget oder die Tasten an deinem Headset gesteuert werden kann. Du kannst alle Songs nach Titel oder Künstler sortieren und den Equalizer anpassen. Das Filtern nach Künstler oder Album wird noch nicht unterstützt, bleib dran!
         
-        Die App beinhaltet ein Widget mit anpassbarer Textfarbe, Transparenz und Hintergrundfarbe.
+        Die Anwendung beinhaltet ein Widget mit anpassbarer Textfarbe, Transparenz und Hintergrundfarbe.
         
         Beinhaltet keine Werbung oder unnötige Berechtigungen. Sie ist komplett Open Source, alle verwendeten Farben sind anpassbar.
 
-        Diese App ist nur eine aus einer größeren Serie von schlichten Apps. Der Rest davon findet sich auf http://www.simplemobiletools.com
+        Diese Anwendung ist nur eine aus einer größeren Serie. Der Rest davon findet sich auf http://www.simplemobiletools.com
```
Comment (tibbi) [inline app/src/main/res/values-de/strings.xml]: are you sure Anwendung is a better word? Translators dont really translate it like that.

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Schlichter Musik-Player</string>
+    <string name="app_name">Simple Music Player</string>
```
Comment (tibbi) [inline app/src/main/res/values-de/strings.xml]: Why are you reverting app name translations?

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (GuepardoApps) [reply app/src/main/res/values-de/strings.xml]: As App is short for Application, I think "Anwendung" fits better. But you can use both. App is just an anglicism.

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (GuepardoApps) [reply app/src/main/res/values-de/strings.xml]: It's the application name, and I think it's like a persons name. You would not translate it either. But also just a suggestion.

Comment (tibbi) [reply app/src/main/res/values-de/strings.xml]: app names should be translated too

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (tibbi) [reply app/src/main/res/values-de/strings.xml]: ye I want to keep it, I like those current german translations

**Pull request closed without merging**